traefik/config.json

```diff
@@ -31,6 +31,7 @@
       "enabled": false,
       "resolvers": []
     },
+    "pilot_token": "",
```

Comment:
Almost ready to merge, please remove this line. Everything within `options` will be set as default. We don't want users to have this unknown option as a default.